test/__init__.py

```diff
@@ -25,7 +25,7 @@ class NativeResourceTest(unittest.TestCase):
 
     def setUp(self):
         NativeResource._track_lifetime = True
-        #init_logging(LogLevel.Trace, 'stderr')
+        # init_logging(LogLevel.Trace, 'stderr')
```

Comment:
Extremely minor nitpick: Remove?

Reply:
This was here when I originally saw it and I think it's left in here for the purpose of our own internal testing. The change reflected here is just the addition of that space between the content and the line =P